themes/foulab-2018/static/js/status.js

```diff
@@ -1,12 +1,9 @@
-function handleStatus() {
-    if (JSON.parse(this.responseText)["status"] === "OPEN") {
-	document.getElementById("status").style.visibility = "visible";
-    }
-}
-
-document.addEventListener("DOMContentLoaded", function(event) {
-    var a = new XMLHttpRequest();
-    a.addEventListener("load", handleStatus);
-    a.open("GET", "https://foulab.org/status/");
-    a.send();
-});
+// Fetch the status of the lab from the server and display it.
+// Thank you modern javascript for making this so easy!
+fetch("https://foulab.org/status/")
```

Comment:
Definitely can be, yes. It doesn't work locally due to CORS anyways, so relative or not doesn't make much difference. I'll change it.

Reply:
Looks like this has been there since inception: https://github.com/FOULAB/foulab-dot-org-hugo/blob/45b7ad806fdce3216f499403b71c69d98183f87f/themes/foulab-2018/static/js/status.js#L10

I think absolute might help with local testing, because if the lab happens to the open, it exercises the code path and makes the OPEN element show.

I think you're right CORS might be breaking this right now. I'm happy to weak the CORS headers on /status/ to allow any origin fetch it.